app/models/category.rb

```diff
@@ -1,3 +1,7 @@
 class Category < ActiveRecord::Base
   has_many :items
+
+  def self.all_names
```

Comment:
This looks like something that is going to get used for a select menu somewhere... is that right? If so, there are some handy rails things we can use for that.